extensions/sql-database-projects/package.json

```diff
@@ -268,6 +273,10 @@
           "command": "sqlDatabaseProjects.editProjectFile",
           "when": "false"
         },
+        {
+          "command": "sqlDatabaseProjects.addSqlBinding",
+          "when": "editorLangId == csharp && !azdataAvailable && resourceExtname"
```

Comment:
`&& resourceScheme != untitled` would be better I think - files don't necessarily have to have extensions.